lib/rubocop/rspec/top_level_group.rb

```diff
@@ -10,11 +10,21 @@ module TopLevelGroup
       def_node_matcher :example_or_shared_group?,
                        (ExampleGroups::ALL + SharedGroups::ALL).block_pattern
 
-      def on_block(node)
-        return unless respond_to?(:on_top_level_group)
-        return unless top_level_group?(node)
+      def on_new_investigation
+        top_level_groups.each do |node|
+          if respond_to?(:on_top_level_example_group)
+            example_group?(node, &method(:on_top_level_example_group))
```

Comment:
and we can have the same for other methods like describe

Reply:
Yep, if we'll eventually need them.

Initially, I was going to add code that would call `on_top_level_shared_group`, but that would only count one usage, so decided not to add it just now.